lib/tsdoc.json

```diff
@@ -2,5 +2,11 @@
 	"$schema": "https://developer.microsoft.com/json-schemas/tsdoc/v0/tsdoc.schema.json",
 	"extends": [
 		"typedoc/tsdoc.json"
+	],
+	"tagDefinitions": [
+		{
+		"tagName": "@groupDescription",
+		"syntaxKind": "block"
+		}
```

Comment:
This is not needed in [typedoc 0.25.8](https://github.com/TypeStrong/typedoc/releases/tag/v0.25.8), which is the version in the `main` branch.

Reply:
removed